libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -125,8 +127,14 @@ impl RestResourceProvider for BiomeRestResourceManager {
                 ));
                 resources.push(make_login_route(
                     credentials_store.clone(),
+                    self.refresh_token_store.clone(),
+                    self.rest_config.clone(),
+                    Arc::new(AccessTokenIssuer::new(
+                        self.token_secret_manager.clone(),
+                        self.refresh_token_secret_manager.clone(),
```

Comment:
Also need to add a feature guard for this, which includes the biome-refresh-tokens

Reply:
I don't think adding the feature guard here is necessary since, the entire match statement is feature guarded

Reply:
Ok, didn't realize the refresh token secret manager isn't necessarily behind the refresh-tokens feature.